web/site/js/vis/Node.js

```diff
@@ -105,41 +105,41 @@ export default class Node extends VObject {
         failedMask.scale.set(0.5, 0.5);
         failedMask.visible = false;
         this._root.addChild(failedMask);
-        this._failedMask = failedMask
+        this._failedMask = failedMask;
```

Comment:
the style change seems unrelated to this PR. 

We currently don't have style enforcement on javascript. So, maybe better to enforce the style using a lint tool with a separate PR. 

Thoughts?

Reply:
Yes, I agree. I got carried away with these semicolons, but you are right. I'm not going to make the PR for this style problem, but I really this it is unnecessary to leave lines without semicolons in JavaScript.

Reply:
> Yes, I agree. I got carried away with these semicolons, but you are right. I'm not going to make the PR for this style problem, but I really this it is unnecessary to leave lines without semicolons in JavaScript.

my javascript editor did this when formatting, not sure why though.

Reply:
Oh, yeah. It seems fine, I'm not a JS specialist to criticize anything. I was just expressing an internal rage of JS crazy rules. I didn't mean anything.